evm_loader/cli/src/errors.rs

```diff
@@ -0,0 +1,56 @@
+//! Error types
+#![allow(clippy::use_self)]
+
+use num_derive::FromPrimitive;
+use solana_program::{decode_error::DecodeError, program_error::ProgramError};
+use thiserror::Error;
+
+/// Errors that may be returned by the neon-cli program.
+#[derive(Clone, Debug, Eq, Error, FromPrimitive, PartialEq)]
```

Comment:
You don't need to derive "FromPrimitive" here.

Reply:
Fixed